parm/use_cases/model_applications/convection_allowing_models/EnsembleStat_fcstHRRR_fcstOnly_SurrogateSevere.conf

```diff
@@ -110,3 +135,7 @@ FCST_PCP_COMBINE_OUTPUT_TEMPLATE = {init?fmt=%Y%m%d}/hrrr_ncep_{init?fmt=%Y%m%d%
 
```

Comment:
The case study data for date 20200205 does not exist in /d1/METPlus_Data on eyewall.